### PR TITLE
Remove deprecated functionality targeted to 1.9

### PR DESCRIPTION
**Pull request by mramato**
Since I've been lazy on CHANGES lately, I also updated it with info from recent pull requests. There's still one deprecated piece of functionality targeted at 1.9 (PolygonGraphics.positions), but removing it is more involved and I'll eventually open a separate pull for it.

### CODE REVIEW

File: Source/Scene/ModelAnimationCollection.js [hunk 1]
```diff
@@ -100,7 +100,7 @@ define([
      * @param {ModelAnimationLoop} [options.loop=ModelAnimationLoop.NONE] Determines if and how the animation is looped.
      * @returns {ModelAnimation} The animation that was added to the collection.
      *
-     * @exception {DeveloperError} Animations are not loaded.  Wait for the {@link Model#readyToRender} event.
+     * @exception {DeveloperError} Animations are not loaded.  Wait for the {@link Model#readtPromise} to resolve.
```
Comment (pjcozzi) [inline Source/Scene/ModelAnimationCollection.js]: Typo here and throughout, `Model#readtPromise`.  I would have just fixed it, but I wanted to ask if the `{@link Model#readtPromise}` markup in the `DeveloperError` below resolves to a link in our exception display?

File: CHANGES.md [hunk 2]
```diff
@@ -3,10 +3,18 @@ Change Log
 
 ### 1.9 - 2015-05-01
 * Breaking changes
-  *
+  * Removed `ColorMaterialProperty.fromColor`. Pass a `Color` directly to the `ColorMaterialProperty` constructor instead.
```
Comment (pjcozzi) [inline CHANGES.md]: We usually say what version something was deprecated in, e.g.,
> Removed `Camera.setTransform`, which was deprecated in Cesium 1.6. Use `Camera.lookAtTransform`.

I don't know that it is strictly necessary but it does give the impression that we are not just making breaking changes, especially for new users that might not know our deprecation policy.

**Comment by pjcozzi** [comment]
Fixes #2452

File: CHANGES.md (hunk 2 above)
Comment (mramato) [inline CHANGES.md]: I partially agree with you, but I think on the whole we need a firmer deprecation policy.  If it's always 3 releases, we can just say it once at the top, i.e. "Breaking changes that were previously deprecated in 1.6" or something like that.  For now, I'll just add it to each item.

File: Source/Scene/ModelAnimationCollection.js (hunk 1 above)
Comment (mramato) [inline Source/Scene/ModelAnimationCollection.js]: It doesn't, that's just bad coding on my part.  I'll fix it.

**Comment by mramato** [comment]
Ready for another look.

**Pull request merged**